README.md

```diff
@@ -381,8 +381,15 @@ Every new transaction goes on **the top of the list** and should have an enclosi
 
 1. Do your best to make the [provided E2E tests](cypress/e2e/test.cy.js) pass. Check out [this tutorial](https://help.alvalabs.io/en/articles/9028831-how-to-work-with-cypress) to learn how to execute these tests and analyze the results.
 2. Keep the server data in memory.
-
 <!--TASK_INSTRUCTIONS_END-->
+
+### GitHub Codespaces CORS issue
```

Comment:
@Ostter @ludvig @ehsan-alva  I added it here but do let me know if you feel if this should go in a different section.

Reply:
Our goal is to move as much as possible of the instructions/readme to help articles, would it be possible to have this type of information in a help article? Dont we have a help article specifically for code space setup?